src/__tests__/__snapshots__/node.test.ts.snap

```diff
@@ -62,8 +62,8 @@ Object {
       "cover": "cover",
     },
     "borderColor": Object {
+      "DEFAULT": undefined,
```

Comment:
does that come from a breaking change in tailwind? is there any implication for our apps?